phpBB/phpbb/path_helper.php

```diff
@@ -496,4 +496,17 @@ public function get_valid_page($page, $mod_rewrite = false)
 
 		return $page;
 	}
+
+	/**
+	 * Tells if the router is currently in use (if the current page is a route or note)
+	 *
+	 * @return bool
+	 */
+	public function is_router_used()
+	{
+		// Script name URI (e.g. phpBB/app.php)
+		$script_name = $this->symfony_request->getScriptName();
+
+		return basename($script_name) === 'app.' . $this->php_ext;
```

Comment:
Does this also work when the .htaccess redirection is in place?
Bugs that are connected to this are nasty.

Reply:
yes it works. The script name should always be defined

Reply:
Surely it's always defined but does it also always point to the app.php when routing through it but is redirected by .htaccess?

Reply:
yep